examples/dna_language_models/dna_lm.ipynb

```diff
@@ -0,0 +1,1369 @@
+       "                  (lora_A): ModuleDict(\n",
+       "                    (default): Linear(in_features=768, out_features=8, bias=False)\n",
+       "                  )\n",
+       "                  (lora_B): ModuleDict(\n",
+       "                    (default): Linear(in_features=8, out_features=768, bias=False)\n",
+       "                  )\n",
+       "                  (lora_embedding_A): ParameterDict()\n",
+       "                  (lora_embedding_B): ParameterDict()\n",
+       "                )\n",
+       "                (dropout): Dropout(p=0.1, inplace=False)\n",
+       "              )\n",
+       "              (output): BertSelfOutput(\n",
+       "                (dense): Linear(in_features=768, out_features=768, bias=True)\n",
+       "                (LayerNorm): LayerNorm((768,), eps=1e-12, elementwise_affine=True)\n",
+       "                (dropout): Dropout(p=0.1, inplace=False)\n",
+       "              )\n",
+       "            )\n",
+       "            (intermediate): BertIntermediate(\n",
+       "              (dense): Linear(in_features=768, out_features=3072, bias=True)\n",
+       "              (intermediate_act_fn): GELUActivation()\n",
+       "            )\n",
+       "            (output): BertOutput(\n",
+       "              (dense): Linear(in_features=3072, out_features=768, bias=True)\n",
+       "              (LayerNorm): LayerNorm((768,), eps=1e-12, elementwise_affine=True)\n",
+       "              (dropout): Dropout(p=0.1, inplace=False)\n",
+       "            )\n",
+       "          )\n",
+       "        )\n",
+       "      )\n",
+       "    )\n",
+       "    (cls): BertOnlyMLMHead(\n",
+       "      (predictions): BertLMPredictionHead(\n",
+       "        (transform): BertPredictionHeadTransform(\n",
+       "          (dense): Linear(in_features=768, out_features=768, bias=True)\n",
+       "          (transform_act_fn): GELUActivation()\n",
+       "          (LayerNorm): LayerNorm((768,), eps=1e-12, elementwise_affine=True)\n",
+       "        )\n",
+       "        (decoder): Linear(in_features=768, out_features=5504, bias=True)\n",
+       "      )\n",
+       "    )\n",
+       "  )\n",
+       "  (classifier): Linear(in_features=768, out_features=2, bias=True)\n",
+       ")"
+      ]
+     },
+     "execution_count": 115,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Number of classes for your classification task\n",
+    "num_labels = 2\n",
+    "classification_model = DNA_LM(lm, num_labels)\n",
+    "classification_model.to('cuda')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 116,
+   "id": "6c223937-86ea-42ef-991a-050f23b21ef9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from peft import LoraConfig, TaskType\n",
+    "\n",
+    "peft_config = LoraConfig(\n",
+    "    #task_type=\"SEQ_CLS\",\n",
+    "    r=8,\n",
+    "    lora_alpha=32,\n",
```

Comment:
Typically, alpha is ~ 2x rank. Did you test that and found that 32 works better?

Reply:
I haven't experimented with the review comments regarding changing hyperparameters for obtaining a higher score, are these necessary for a tutorial notebook?

Reply:
No, not necessary at all. It's just that when an example deviates from the defaults, I wonder if the author made these choices because they found them to work better or if there is another reason. Same with the dropout rate below.